.github/ISSUE_TEMPLATES/bug_report.yml

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Report bugs and errors found while using the SDK.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+### Your environment
+
+<!-- Version of the SDK when the error occurred -->
+SDK Version:
+
+<!-- What version of the Connect server are you running?
+You can get this information from the Integrations section in 1Password
+https://1password.com/integrations/active
+-->
+Connect Server Version:
+
+<!-- What OS was the SDK running on? -->
+OS:
+
+<!-- What version of Node and TypeScript were you running when the error occurred? -->
+Node Version:
+
+TypeScript Version:
```

Comment:
How often would we need the TypeScript version? I am considering the option to just leave it out and ask for it if it is relevant for the submitted issue. This lowers the friction of submitting a bug (I wouldn't even know where to find this without Googling), while still addressing all relevant cases. What do you think?
```suggestion
<!-- What version of Node were you running when the error occurred? -->
Node Version:
```

Reply:
I added it proactively, but I think you're correct that we won't be too concerned about the TS version. 

I realize now that we distribute the package pre-compiled so it shouldn't be much of a concern unless the bug report comes from someone extending the SDK.